framework/src/actions/CreateExecutionerAction.C

```diff
@@ -110,6 +110,7 @@ CreateExecutionerAction::act()
   Moose::setup_perf_log.pop("Create Executioner","Setup");
 
   _awh.executioner() = executioner;
```

Comment:
That's fine, I much prefer making two independent changes then one that requires application patches immediately. Just leave the ticket open and we'll adjust the AWH later to remove the other reference.